elasticsearch/init/0500-remove-index-patterns-without-uid

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script removes all kibana index patterns to resolve:
+#    https://bugzilla.redhat.com/show_bug.cgi?id=1622822
+# which removes all index patterns that are of the format:
+#    project.<projectname>.*
+#
+set -e
+
+if [ -n "${SDEBUG:-}" ] ; then
+  set -x
+fi
+
+source "logging"
+
+script=$(basename $0)
+size=100
+
+info "Starting init script: ${script}"
+
+function get_scroll_id(){
+  echo $1 | python -c  '
+import json
+import sys
+resp = json.load(sys.stdin)
+print resp["_scroll_id"]
+'
+}
+
+function get_errors(){
+  echo $1 | python -c  '
+import json
+import sys
+resp = json.load(sys.stdin)
+print resp["errors"]
+'
+}
+
+function get_hits(){
+  echo $1 | python -c  '
+import json
+import sys
+try:
+  resp = json.load(sys.stdin)
+  print resp["hits"]["total"]
+except:
+  print True
+'
+}
+
+function get_delete_payload(){
+echo $1 | python -c  "
+import json
+import sys
+import re
+resp = json.load(sys.stdin)
+pattern = re.compile('^project\.([a-zA-Z0-9\-]*)?\.\*$')
```

Comment:
`^project\.([a-zA-Z0-9][a-zA-Z0-9\-]*)?\.\*$'
that is, it must begin with a non-dash - also, why the `?` - this means you will match on `project..*`

Reply:
These are the patterns I tested which resulted in what I expected.  Maybe its adjustable but when running against the new code I want to ensure we do not remove them.  Possibly we could resolve with a 'history' file that skips running all together but that does not satisfy the idempotentcy we desire
 
![image](https://user-images.githubusercontent.com/4548408/45159316-6b772480-b1b4-11e8-9b15-bcf983c0110e.png)

Reply:
Removing the `?` but still matches on `project..*` which is acceptable as I believe that's  match we should remove anyway